openwisp_monitoring/monitoring/configuration.py

```diff
@@ -243,7 +243,7 @@ def _get_access_tech():
                     _('Total download traffic'),
                     _('Total upload traffic'),
                 ],
-                'unit': _(' GB'),
+                'unit': 'adaptive_bytes',
```

Comment:
reminder: update help text (unit of measure is now adaptive so we have to account for this)